.gitpod.yml

```diff
@@ -1,5 +1,29 @@
+image:
+  file: .gitpod.Dockerfile
+# Options to prebuild the image after github events and set notifications/badges
+github:
+  prebuilds:
+    # enable for the master/default branch (defaults to true)
```

Comment:
I think we should just keep options that are actually needed and link to docs with full list of options, comments, etc.
It also looks like we are going a bit over the top here. Why have both `addComment` and `addBadge` enabled  for example? Or why pre-build non-PR branches?